src/dmd/backend/evalu8.d

```diff
@@ -1876,4 +1898,51 @@ elem * evalu8(elem *e, goal_t goal)
   return e;
 }
 
-#endif /* !SPP */
+/******************************
+ * This is the same as the one in el.c, but uses native D reals
+ * instead of the soft long double ones.
+ */
+
+extern (D) real el_toldoubled(elem *e)
```

Comment:
Sorry, still more uses of `real` instead of `targ_ldouble`...

Reply:
fixed